src/journal-remote/journal-remote.c

```diff
@@ -144,6 +144,7 @@ static int spawn_curl(const char* url) {
                                 "-HAccept: application/vnd.fdo.journal",
                                 "--silent",
                                 "--show-error",
+                                "--no-buffer",
```

Comment:
Good question. This certainly could be conditional and as it is it will change the --url case where it is not using the --follow option. I didn't really feel strongly either way on whether or not the non --follow case should have it which is why I left it as unconditional. With this we will be able to process data sooner in all cases but we may end up popping out of the epoll in the event loop more often. I'll defer to your judgment as to whether I should go back and make this conditional.